ebpf-io/ebpf.io-website
# chore: dependencies update 01.24

This pull request brings dependencies update

**Steps to check**:

- open [preview](https://deploy-preview-555--ebpfio.netlify.app/)
- make sure that everything works as expected
- open project in development mode and make sure that everything okay
